11ty/definitions/black-asian-minority-ethnic.md

```diff
@@ -1,5 +1,29 @@
 ---
 title: Black Asian Minority Ethnic (BAME)
 slug: black-asian-minority-ethnic
-defined: false
----
\ No newline at end of file
+flag:
+  level: avoid
+  text: 'Avoid'
+defined: true
+excerpt: umbrella term encompassing all people from Black, Asian and minoritised ethnic groups living in the UK
+speech: adjective
+reading:
+  - text: 'Race and identity: ‘Don’t call me BAME’'
+    href: https://www.bbc.co.uk/news/av/uk-53205008
+  - text: 'Racial Categorisation and Terminology'
+    href: https://blackbritishacademics.co.uk/about/racial-categorisation-and-terminology/
+---
+umbrella term encompassing all people from Black, Asian and minoritised ethnic groups living in the UK
+
+## Issues
+
+BAME can be near meaningless as it groups together people who have different cultures and experiences of inequality and racism. "Minority" is also a problematic word for describing peoples because, for example, it (inaccurately) connotes powerlessness and subordination (see [minorities](/definitions/minorities/)).
+
+## Usage Tip
+
+Be more specific wherever possible. Are you referring to Black people? British Pakistanis? Roma? Ask individuals how they identify themselves and use their terms.
+
+## Read also
+
+- People of Color
```

Comment:
These are great links! Generally we try to only link things once the terms have been defined.

Reply:
@tatianamac Hey, I'm a British Asian minority and the definition: 'umbrella term encompassing all people from Black, Asian and minoritised ethnic groups living in the UK' sounds about right to me 👍🏼